publication-event-dtopublicationdoi-producer/build.gradle

```diff
@@ -0,0 +1,23 @@
+def log4jVersion = "2.13.2"
+
+dependencies {
+    implementation project(":publication-doi-commons")
+
+    implementation group: 'com.github.BIBSYSDEV.nva-datacite-mds', name: 'event-handler-template', version: '087d568dda7adf671c843d5c0294a4d7ff54e756'
```

Comment:
Any releases for this yet?

Reply:
Unsure. We can probably make one? @axthosarouris

Reply:
Fixed by using a pre-release.